tools/sgcloudrun/config.go

```diff
@@ -370,3 +377,15 @@ func fetchImpersonatedAccessToken(ctx context.Context, serviceAccountEmail strin
 
 	return tokens.AccessToken, nil
 }
+
+//nolint:gochecknoglobals
+var lower = []rune("abcdefghijklmnopqrstuvwxyz")
```

Comment:
could define a private function returning this and it'll get inlined anyway?

Reply:
did this instead:
```
func randomLower(n uint32) string {
	b := make([]rune, n)
	for i := range b {
		// NOTE: code of 'a' is 97, code of 'z' is 122.
		//       so rand.IntN(26) + 97 is code of a "random" lowercase rune.
		//nolint:gosec // we don't need a secure randomizer for this.
		b[i] = rune(rand.IntN(26) + 97)
	}
	return string(b)
}```